src/components/TopBar.jsx

```diff
@@ -0,0 +1,275 @@
+
+            <DropdownMenuSeparator />
+
+            <DropdownMenuGroup>
+              <DropdownMenuLabel>TRELLO</DropdownMenuLabel>
+              <DropdownMenuItem>Profile and visibility</DropdownMenuItem>
+              <DropdownMenuItem>Activity</DropdownMenuItem>
+              <DropdownMenuItem>Cards</DropdownMenuItem>
+              <DropdownMenuItem>Settings</DropdownMenuItem>
+              <DropdownMenuSub>
+                <DropdownMenuSubTrigger
+                  className={cn("flex items-center gap-2")}
+                >
+                  <Sun className="h-4 w-4 rotate-0 scale-100 transition-all dark:-rotate-90 dark:scale-0" />
+                  <Moon className="absolute h-4 w-4 rotate-90 scale-0 transition-all dark:rotate-0 dark:scale-100" />
+                  <span className="sr-only">Toggle theme</span>
+                  <span>Theme</span>
+                </DropdownMenuSubTrigger>
+                <DropdownMenuPortal>
+                  <DropdownMenuSubContent>
+                    <DropdownMenuItem
+                      onClick={() => setTheme("light")}
+                      className={cn("flex items-center gap-2")}
+                    >
+                      <Sun className="h-4 w-4" />
+                      <span>Light</span>
+                    </DropdownMenuItem>
+                    <DropdownMenuItem
+                      onClick={() => setTheme("dark")}
+                      className={cn("flex items-center gap-2")}
+                    >
+                      <Moon className="h-4 w-4" />
+                      <span>Dark</span>
+                    </DropdownMenuItem>
+                    <DropdownMenuItem
+                      onClick={() => setTheme("system")}
+                      className={cn("flex items-center gap-2")}
+                    >
+                      <Palette className="h-4 w-4" />
+                      <span>System</span>
+                    </DropdownMenuItem>
+                  </DropdownMenuSubContent>
+                </DropdownMenuPortal>
+              </DropdownMenuSub>
+            </DropdownMenuGroup>
+
+            <DropdownMenuSeparator />
+
+            <DropdownMenuGroup>
+              <DropdownMenuItem
+                className={cn("flex items-center justify-between")}
+              >
+                <span>Create Workspace</span>
+                <UsersRound className="h-4 w-4" />
+              </DropdownMenuItem>
+            </DropdownMenuGroup>
+
+            <DropdownMenuSeparator />
+
+            <DropdownMenuGroup>
+              <DropdownMenuItem>Help</DropdownMenuItem>
+              <DropdownMenuItem>Shortcuts</DropdownMenuItem>
+            </DropdownMenuGroup>
+
+            <DropdownMenuSeparator />
+
+            <DropdownMenuGroup>
+              <DropdownMenuItem>Log out</DropdownMenuItem>
+            </DropdownMenuGroup>
+          </DropdownMenuContent>
+        </DropdownMenu>
+      </div>
+    </nav>
+  );
+};
+
+export default TopBar;
```

Comment:
ditto

Reply:
Got it! For that file, it was similar to the previous one. I retained those changes to keep the layout consistent since I’ll need them anyways.